net-irc/quassel/quassel-9999.ebuild

```diff
@@ -58,9 +58,9 @@ GUI_DEPEND="
 	urlpreview? ( dev-qt/qtwebengine:5[widgets] )
 "
 
-DEPEND="
+COMMON_DEPEND="
```

Comment:
Introduction of `COMMON_DEPEND` is unnecessary. Just reverse `RDEPEND`/`DEPEND` inheritance.

Reply:
OK, I was following the pattern I saw in other ebuilds with boost dependency, and assumed that that's the way to do this ;-) I'll adapt to your suggestion and push a new patchset in a bit.